packages/loot-core/src/platform/server/asyncStorage/index.electron.js

```diff
@@ -36,52 +36,42 @@ function _saveStore() {
   }
 }
 
-function getItem(key) {
+export const getItem = function (key) {
   return new Promise(function (resolve) {
     return resolve(store[key]);
   });
-}
+};
 
-function setItem(key, value) {
+export const setItem = function (key, value) {
   store[key] = value;
   return _saveStore();
-}
+};
 
-function removeItem(key) {
+export const removeItem = function (key) {
   delete store[key];
   return _saveStore();
-}
+};
 
-function multiGet(keys) {
+export const multiGet = function (keys) {
   return new Promise(function (resolve) {
     return resolve(
       keys.map(function (key) {
         return [key, store[key]];
       }),
     );
   });
-}
+};
 
-function multiSet(keyValues) {
+export const multiSet = function (keyValues) {
```

Comment:
Why not…

```js
export function multiSet(keyValues) {
```

(in a bunch of places not just here)

Reply:
This is due the that way I thought we could consistently type these APIs: given we have web/electron/... it is suboptimal to type on every file all arguments / return types for each function. We risk having discrepancies. 
And I could not find a way to import an override type for those functions. 
You  can see how it looks in `packages/loot-core/src/platform/exceptions/index.browser.ts`: we can import from `index.d.ts` all types and assign them to the consts without having to rewrite the definition. I feel like it is a decent way  to ensure type safety without duplication but also open to other suggestions

Reply:
Makes sense!